src/ethereum/ethPlugin.ts

```diff
@@ -97,6 +103,14 @@ export class EthereumTools implements EdgeCurrencyTools {
     }
   }
 
+  async checkPublicKey(publicKey: JsonObject): Promise<boolean> {
+    const ethereumPublicKey = asMaybe(asEthereumPublicKey)(publicKey)
+    if (ethereumPublicKey == null) return false
+    // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
+    if (!EthereumUtil.isValidChecksumAddress(ethereumPublicKey)) return false
```

Comment:
Does this pass with all-lowercase addresses? If so, maybe we could comment on that.